### PR TITLE
front: focus on a train with conflict on click in studies

### LINKED ISSUE #5944: Focus on a train with conflict when clicking on the conflict list
### Description and goal

When clicking on the conflict nothing happens:
image

### Acceptance criteria

- [ ] When we click on a conflict, the map is focused on the path of the first train involved in the conflict
- [ ] When we click on a conflict, the GET is focused on first train involved in the conflict

We should focus on train in conflict

### PR DESCRIPTION
**Pull request by RomainValls**
closes #5944 

Clicking on a conflict now : 

highlights the curve of the first train involved in conflict
focus on its train path on the map

### CODE REVIEW

**Review by Akctarus — COMMENTED** [review]
Tested, works well, just left one comment

File: front/src/modules/trainschedule/components/Timetable/Timetable.tsx [hunk 1]
```diff
@@ -133,6 +133,13 @@ export default function Timetable({
     }
   };
 
+  const handleConflictClick = (conflict: Conflict) => {
+    if (conflict.train_ids.length > 0) {
+      const firstTrainId = conflict.train_ids[0];
+      dispatch(updateSelectedTrainId(firstTrainId));
+    }
```
Comment (Akctarus) [inline front/src/modules/trainschedule/components/Timetable/Timetable.tsx:140]:
```suggestion
  const hasTrainIds = conflict.train_ids.length > 0;
  return hasTrainIds ? dispatch(updateSelectedTrainId(conflict.train_ids[0])) : null;
```

Comment (clarani) [reply front/src/modules/trainschedule/components/Timetable/Timetable.tsx:140]: In handleConflictClick, we don't need to return anything but we do do some things. So no need to move everything in a return statement :)
I think it's perfect like this

**Review by clarani — APPROVED** [review]
Tested, it works great !

**Review by SharglutDev — COMMENTED** [review]
I think that you should add a `cursor: "pointer";` on the conflict card so the user knows it's clickable.

It's not in the issue but I thought I heard someone say that we should also change the projection ?

**Review by Akctarus — APPROVED** [review]

**Pull request merged**